src/Geno/C++/GUI/MainWindow.cpp

```diff
@@ -46,6 +46,85 @@
 #define sscanf sscanf_s
 #endif // _WIN32
 
+#if defined( __APPLE__ )
+#define _GLFW_COCOA
+#include <../src/internal.h> // MarcasRealAccount: Dangerous piece of code
```

Comment:
Can be made less ugly if we add `/third_party/` to the include directories. Then we can just write
```cpp
#include <glfw/src/internal.h>
```
Which is still a little hacky, but still a hundred times better than a relative include.